backend/Dockerfile

```diff
@@ -1,8 +1,17 @@
 FROM node:10
+
```

Comment:
Why are you adding empty lines everywhere? The point of empty lines is to group code and commands to be more readable. Adding them everywhere just makes things harder to read.

Reply:
The idea was to make it more readable in code, but i understand it's difficult while reviewing. Will fix it and update right away